docs/guides/theming/WhiteLabeling.stories.mdx

```diff
@@ -64,12 +103,12 @@ const MyApp = ({ children }) => {
 };
 ```
 
-Only by customizing the theme, the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvGlobalActions`, `HvTypography`, and `HvContainer`
-components were able change their default styling.
+By customizing the theme variables, the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvHeaderMenuIte` and `HvHeaderMenuBarBar`
```

Comment:
I believe there are some typos here, maybe:

"By customizing the theme variables,  we are able to change the default styling of the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvHeaderMenuItem`, and `HvHeaderMenuBarBar` components to meet the desired UI guidelines."